hbase-client/src/main/java/org/apache/hadoop/hbase/exceptions/MasterStoppedException.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable
+ * law or agreed to in writing, software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ * for the specific language governing permissions and limitations under the License.
+ */
+
+package org.apache.hadoop.hbase.exceptions;
+
+import org.apache.hadoop.hbase.DoNotRetryIOException;
+import org.apache.yetus.audience.InterfaceAudience;
+
+/**
+ * Thrown when the master is stopped
+ */
+@InterfaceAudience.Private
```

Comment:
Maybe we should declare it as IA.Public? What is the IA annotation for other exceptions under this package?

Reply:
I think we have mixed IA for these classes, but yes I also believe that `MasterStoppedException` should be IA.Public.

Reply:
In general, I think user could catch MasterStoppedException and process it, but we do not expect user to create a MasterStoppedException right? The downside of IA.Public is that, we can not change the constructors of this exception any more...

Maybe we should have a IA.LimitedPrivate for exceptions?

Reply:
We do not construct or catch MasterStoppedException in client side now,  so I use IA.Private. 
And yes, It should be IA.Public, and IA.LimitedPrivate is better